.github/workflows/ci.yaml

```diff
@@ -40,7 +38,20 @@ jobs:
       - name: Run cargo fmt
         run: |
           cargo fmt --all -- --check
-      
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Rustup toolchain install
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: 1.78.0
+          components: rustfmt, clippy
```

Comment:
we don't need `rustfmt` and `clippy` for this job right?

Reply:
good catch